.github/workflows/create-release.yml

```diff
@@ -172,7 +172,7 @@ jobs:
           rm -rf re-version/out/liquibase-${{ needs.setup.outputs.version }}.zip
           IFS=',' read -ra EXT <<< "${{ needs.setup.outputs.extensions }}"
           for i in "${EXT[@]}"; do
-            cp ~/.m2/repository/org/liquibase/ext/$i/${{ needs.setup.outputs.version }}/$i-${{ needs.setup.outputs.version }}.jar re-version/out/liquibase-${{ needs.setup.outputs.version }}/internal/extensions/$i-${{ needs.setup.outputs.version }}.jar || echo "Failed to move $i artifact"
+            cp ~/.m2/repository/org/liquibase/ext/$i/${{ needs.setup.outputs.version }}/$i-${{ needs.setup.outputs.version }}.jar re-version/out/liquibase-${{ needs.setup.outputs.version }}/internal/extensions/$i.jar || echo "Failed to move $i artifact"
```

Comment:
Was this change previously made for the `4.29.1` or `4.29.0` release?

Reply:
> Was this change previously made for the `4.29.1` or `4.29.0` release?

those versions are already in the zip and installer with the version in the jar name - we can't do anything about it here.
But this pr also adds java logic to ignore them going forward so they are not loaded by liquibase.